motor/docstrings.py

```diff
@@ -1122,7 +1066,6 @@ async def add_3_to_x():
     key, if not given :data:`~pymongo.ASCENDING` is assumed
 """
 
-
```

Comment:
Are these blank space changes intentional?

Reply:
Yes. Some had 2 empty lines and others 1, I changed them all to 1.